bin/c8.js

```diff
@@ -40,3 +29,20 @@ function outputReport () {
     omitRelative: argv.omitRelative
   })
 }
+
+(async function run () {
+  if (argv._[0] === 'report') {
+    argv = yargs.parse(process.argv) // support flag arguments after "report".
+    outputReport()
+  } else {
+    if (argv.clean) {
+      await _p(rimraf)(argv.tempDirectory)
```

Comment:
@addaleax @jasnell do we know the timing of when the `fs.promises` API will stop being experimental? Given that this library is exercising various bleeding-edge features, e.g., the Inspector itself, my preference would be for using `fs.promises` rather than `util.promisify`.

Reply:
@bcoe - still `fs.promises` experimental though, can I just stick to at least the stable(r) ones for now? 🤔 So that the only thing we have to worry about right now is the recursive `fs.mkdir` shipping in the next + 1 release.

Reply:
sounds good to me, let's stick with util.promisify ... I'm excited for fs.promises though!

Reply:
fs.promises will likely go stable early in 11.x timeframe (hopefully before end of the year)... We should have at least one module in CITGM that uses it before it can graduate.

Reply:
@jasnell -- good to hear! Happy to move over to `promises` then since `fs.mkdir` is now out of question. I'll update the PR.